ott/core/initializers_lr.py

```diff
@@ -342,17 +342,23 @@ class KMeansInitializer(LRInitializer):
 
   Args:
     rank: Rank of the factorization.
+    min_iterations: Minimum number of iterations.
```

Comment:
maybe explain iterations of kmeans clustering?

Reply:
Thanks, done!